projects/kubernetes-sigs/image-builder/Makefile

```
@@ -49,6 +49,7 @@ ADDITIONAL_PAUSE_1-23_FROM=1-22
 ADDITIONAL_PAUSE_1-24_FROM=1-23
 ADDITIONAL_PAUSE_1-25_FROM=1-24
 ADDITIONAL_PAUSE_1-26_FROM=1-26
```

```suggestion
ADDITIONAL_PAUSE_1-26_FROM=1-25
```

shouldn't this be updated now?